src/main/java/bowling/ui/ResultView.java

```diff
@@ -0,0 +1,61 @@
+package bowling.ui;
+
+import bowling.domain.frame.Frame;
+import bowling.domain.value.FrameNumber;
+import bowling.domain.value.FramePins;
+import bowling.domain.value.Pins;
+import bowling.domain.value.Player;
+
+import java.util.stream.Collectors;
+
+public class ResultView {
+    private static final int START_FRAME = 1;
+    private static final int FINAL_FRAME = 10;
+    private static final String PLAYER_NAME = "NAME";
+    private static final String DELIMITER = "|";
+    private static final String LINE = "|";
+    private static final String ENTER = "\r\n";
+
+    public void printBowlingResult(Frame frame, Player player) {
+        printHead();
+        printBody(frame, player);
+    }
+
+    private void printHead() {
+        StringBuilder bowlingBuilder = new StringBuilder();
+        printPlayerName(bowlingBuilder, PLAYER_NAME);
+
+        for (int i = START_FRAME; i <= FINAL_FRAME; i++) {
+            printFrame(bowlingBuilder, String.valueOf(i));
+        }
+
+        System.out.println(bowlingBuilder.toString());
```

Comment:
toString() 을 해주지 않아도 됩니다 😄 
```suggestion
        System.out.println(bowlingBuilder);
```

Reply:
감사합니다! 바로 반영했습니다 👍